pulumi/volumeReaper/src/cleanup.ts

```diff
@@ -0,0 +1,74 @@
+import k8s from '@kubernetes/client-node'
```

Comment:
don't want to run into same default import error case
```suggestion
import * as k8s from '@kubernetes/client-node'
```

Reply:
Interesting... well at least they all follow the same pattern now I guess haha